requirements/base.txt

```diff
@@ -1,36 +1,80 @@
-asgiref==3.4.1
 argon2-cffi==20.1.0
```

Comment:
I think something happened with the base require file- some of these dependencies such as `autopep8` are development only - because we split up the requirement file you can't `pip freeze` the file anymore